tests/test_step_parsing.py

```diff
@@ -154,3 +156,10 @@ def test_timeouts(timeouts_config):
 def test_bling(example1_config: Config) -> None:
     assert example1_config.steps['run training'].category == "Training"
     assert example1_config.steps['run training'].icon == "https://valohai.com/assets/img/valohai-logo.svg"
+
+
+def test_widget(example1_config: Config) -> None:
+    assert example1_config.steps['run training'].parameters['sql-query'].widget and \
+        example1_config.steps['run training'].parameters['sql-query'].widget.type == "sql"
+    assert example1_config.steps['run training'].parameters['output-alias'].widget and \
+        example1_config.steps['run training'].parameters['output-alias'].widget.type == "datumalias"
```

Comment:
Can you also check that `widget` is `None` as expected for some other parameter?